samples/apps/helloworld/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func main() {
+	http.ListenAndServe(":8080", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think this should get the PORT environment variable to be a correct example.

Reply:
Done